collection-scripts/gather_machineconfig_ondisk

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash 
+
+BASE_COLLECTION_PATH="must-gather"
+MACHINECONFIG_CONFIG_PATH=${OUT:-"${BASE_COLLECTION_PATH}/machine_config_ondisk"}
+
+mkdir -p "${MACHINECONFIG_CONFIG_PATH}"/
+
+# gather_machineconfig_ondisk_currentconfig collects the MCO's "on-disk" MachineConfig files
+function gather_machineconfig_ondisk {
+     
+    echo "INFO: Gathering on-disk MachineConfig from degraded nodes"
+    # we only really need the info from nodes that the MCO has marked as degraded 
+    MACHINECONFIG_DEGRADED_NODES=$(oc get node -o jsonpath='{.items[?(@.metadata.annotations.machineconfiguration\.openshift\.io/state=="Degraded")].metadata.name}')
```

Comment:
I think...I can tell it to collect only if the `kube-system/bootstrap` configmap is absent?

I do worry slightly about making this too narrow because I can't anticipate all the bugs we'll get and what signals will be "dependable" in those scenarios.

Reply:
Are we moving forward with this suggestion (is the PR still in draft)?